site/app/templates/grading/electronic/DiscussionForumPanel.twig

```diff
@@ -1,19 +1,9 @@
 {% import _self as self %}
+{{ add_twig_module_js('grading/electronic/DiscussionForumPanel.js') }}
 <div id="discussion_browser" class="draggable rubric_panel" style="display:block;">
-        <div class="draggable_content" style="width:100%">
-        	<div id="posts_list" style="padding:20px;margin-top:10px;max-height: 100%; overflow: initial;" >
-            	{{ discussion_forum_content|raw }}
-            </div>
-        </div>
-    
-</div>
-
-<script>
-
-	
-		$('#posts_list').css('min-width', '-webkit-fill-available');
-		$(function() {
-			generateCodeMirrorBlocks(document);
-		});
-
-</script>
\ No newline at end of file
+	<div class="draggable_content" style="width:100%">
+		<div id="posts_list" style="padding:20px;margin-top:10px;max-height: 100%; overflow: initial;" >
+			{{ discussion_forum_content|raw }}
```

Comment:
`discussion_forum_content` contains raw HTML that would get parsed by the .twig file - I think it would be better to move the HTML that is generated over to Twig instead, but it might be better for a different PR since it needs a refactor in the PHP code.